doc/man1/openssl.pod

```diff
@@ -554,6 +562,36 @@ Read the password from standard input.
 
 =back
 
+=head2 Random State Options
+
+Prior to OpenSSL 3.0, it was common for applications to store information
+about the state of the random-number generator in a file that was loaded
+at startup and rewritten upon exit. On modern operating systems, this is
+generally no longer necessary as OpenSSL will seed itself from the
+appropriate CPU flags, device files, and so on. These flags are still
+supported for special platforms or circumstances that might require them.
+
+It is generally an error to use the same seed file more than once and
+every use of B<-rand> should be paired with B<-writerand>.
+
+=over 4
+
+=item B<-rand> I<files>
+
+A file or files containing random data used to seed the random number
+generator.
+Multiple files can be specified separated by an OS-dependent character.
+The separator is B<;> for MS-Windows, B<,> for OpenVMS, and B<:> for
```

Comment:
Hmmm, not sure `B<` is right in this case...  `C<`?  Frankly, I'm not entirely sure.  @t8m, do you have an opinion?

Reply:
I would vote for `C<` as well.

Reply:
@richsalz Please change this and I'll approve.

Reply:
Amended commit pushed.  Is there guidance anywhere on when to use ```C<>``` vs ```B<>```?  We're not consistent, so it will be will-o-the-wisp, which is okay I guess.

I would prefer all commits are squashed when merging.